konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/build/extension/ServeBuildCommand.java

```diff
@@ -0,0 +1,174 @@
+                out.format("Invalid configuration defined by: %s", configuration);
+                System.exit(1);
+            } else {
+                FileUtils.writeStringToFile(savePath, jsonConfiguration.get("pipeline").toString(), StandardCharsets.UTF_8);
+
+                Class<?> buildCliClass = Class.forName("ai.konduit.serving.build.cli.BuildCLI");
+                Object buildCli = buildCliClass.getConstructor().newInstance();
+                Method buildCliMainMethod = buildCliClass.getMethod("exec", String[].class);
+                buildCliMainMethod.setAccessible(true);
+
+                Profile profile = profileName != null ? ProfileCommand.getProfile(profileName) : ProfileCommand.getDefaultProfile();
+                if(profile == null) {
+                    if(profileName == null) {
+                        out.println("Couldn't find a default profile. Starting server with 'CPU' profile");
+                    } else {
+                        out.format("Couldn't find a profile with the specified name: '%s'. Starting server with 'CPU' profile.%n", profileName);
+                    }
+
+                    profile = ProfileCommand.getProfile("CPU");
+                }
+
+                List<String> args = new ArrayList<>();
+                args.add("-p"); args.add(savePath.getAbsolutePath());
+                args.add("-c"); args.add(String.format("classpath.outputFile=%s", mfJar.getAbsolutePath())); args.add("classpath.type=JAR_MANIFEST");
+                args.add("-dt"); args.add("CLASSPATH");
+                args.add("-d"); args.add(profile.computeDevice());
+                args.add("-a"); args.add(profile.cpuArchitecture());
+                args.add("-o"); args.add(profile.operatingSystem());
+
+                if(profile.serverTypes() != null) {
+                    for(String serverType : profile.serverTypes()) {
+                        args.add("-s"); args.add(serverType);
+                    }
+                }
+
+                if(profile.additionalDependencies() != null) {
+                    for(String additionalDependency : profile.additionalDependencies()) {
+                        args.add("-ad"); args.add(additionalDependency);
+                    }
+                }
+
+                buildCliMainMethod.invoke(buildCli, (Object) args.toArray(new String[args.size()]));
+
+                if (Strings.isNullOrEmpty(this.classpath)) {
+                    this.classpath = mfJar.getAbsolutePath();
+                } else {
+                    this.classpath += File.pathSeparator + mfJar.getAbsolutePath();
+                }
+            }
+        } catch (IOException e) {
+            log.error("Unable to write build pipeline data to {}.", savePath.getAbsolutePath(), e);
+            System.exit(1);
+        } catch (ClassNotFoundException e) {
+            out.println("Unable to find classes for building manifest jar. Continuing without runtime build...");
```

Comment:
If the manifest JAR build fails, we likely can't launch the server successfully (due to missing modules/dependencies), so probably should just exit.